modules/siddhi-core/src/test/java/io/siddhi/core/query/function/FunctionTestCase.java

```diff
@@ -317,4 +317,164 @@ public void testFunctionQuery2() throws InterruptedException {
         siddhiManager.createSiddhiAppRuntime(cseEventStream + query);
     }
 
+    @Test(expectedExceptions = SiddhiAppCreationException.class)
+    public void testFunctionQuery9() throws InterruptedException {
+        log.info("CreateSet Test1");
+
+        SiddhiManager siddhiManager = new SiddhiManager();
+
+        String planName = "@app:name('createSet') ";
+        String cseEventStream = "define stream fooStream (symbol string, deviceId long);";
+        String query = "@info(name = 'query1') " +
+                "from fooStream " +
+                "select createSet(symbol, deviceId) as initialSet  " +
+                "insert into barStream;";
+
+        siddhiManager.createSiddhiAppRuntime(cseEventStream + query);
+    }
+
+    @Test
+    public void testFunctionQuery10() throws InterruptedException {
```

Comment:
If you are writing a test case, then it is necessary to have a Assert statement or expectedExceptions. Please make sure, you have fixed similar issues in other places...